### ISSUE TITLE
Lab 7 issues

### ISSUE THREAD

**Issue by sv720** [MatchLab-Imperial/deep-learning-course#5]
"from keras.layers import Dropout, concatenate, merge, UpSampling2D, MaxPooling2D"
merge not supported for Keras 2+ 
Inclusion of merge prevents from running
replace with: 

"from keras.layers import Dropout, concatenate, UpSampling2D, MaxPooling2D"

**Comment by sv720** [comment]
More library incompatibilities and fixes: 

#from keras.layers.normalization import BatchNormalization
from tensorflow.keras.layers import BatchNormalization
#from keras.layers.advanced_activations import LeakyReLU
from keras.layers import LeakyReLU

**Comment by TomTomTommi** [comment]
Done
